packages/plugin-express/src/express.js

```diff
@@ -33,20 +29,11 @@ module.exports = {
 
       if (!client._config.autoDetectErrors) return next()
```

Comment:
This isn't correct anymore — we have to run `next` in the context otherwise APIs like `Bugsnag.addMetadata` will be global instead of request scoped

Reply:
In fact, isn't this check completely unnecessary now? Unhandled errors will bubble up to the global exception handler, which checks for this itself